library/core/src/result.rs

```diff
@@ -1095,10 +1057,8 @@ impl<T: Default, E> Result<T, E> {
     /// assert_eq!(0, bad_year);
     /// ```
     ///
-    /// [`parse`]: ../../std/primitive.str.html#method.parse
+    /// [`parse`]: str::parse
     /// [`FromStr`]: ../../std/str/trait.FromStr.html
```

Comment:
Had to leave this because can't link from `core` to `std` yet.

Reply:
`FromStr` is in core, so you can use
```suggestion
    /// [`FromStr`]: crate::str::FromStr
```